Sources/Common/DataModel/BoundingBox/index.d.ts

```diff
@@ -588,6 +588,14 @@ declare class BoundingBox {
    */
   intersects(bounds: Bounds, bBounds: Bounds): boolean;
 
+  /**
+   * Intersect box with line and return the parametric values and points of the two intercepts
+   * @param bounds 
+   * @param p1 
+   * @param p2 
+   */
+  intersectBoundingBoxWithLine(bounds: Bounds, p1: Vector3, p2: Vector3) : object;
```

Comment:
Yes, but it returns the parametric values of the intercept points

Reply:
Fair enough, then you need instead to look at [vtkBox.IntersectWithLine](https://gitlab.kitware.com/vtk/vtk/-/blob/master/Common/DataModel/vtkBox.h?ref_type=heads#L103).
Would you mind adding such method in `vtkBox` instead of `vtkBoundingBox` (in order to keep the API as similar to VTK as possible?)

Reply:
Done